packages/control-property-editor/package.json

```
@@ -39,7 +39,7 @@
         "@types/react": "16.14.55",
         "body-parser": "1.20.3",
         "eslint-plugin-react": "7.33.2",
-        "http-proxy-middleware": "2.0.7",
+        "http-proxy-middleware": "2.0.8",
```

```suggestion
        "http-proxy-middleware": "2.0.9",
```
Upgrade needs to be to 9 like the others.